test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -113,11 +139,13 @@ public void AutoFitRegressionTest()
             var validationData = context.Data.TakeRows(trainData, 20);
             trainData = context.Data.SkipRows(trainData, 20);
             var result = context.Auto()
-                .CreateRegressionExperiment(0)
+                .CreateRegressionExperiment(experimentTime)
                 .Execute(trainData, validationData,
                     new ColumnInformation() { LabelColumnName = DatasetUtil.MlNetGeneratedRegressionLabel });
 
             Assert.True(result.RunDetails.Max(i => i.ValidationMetrics.RSquared > 0.9));
```

Comment:
Can we add an assert that we completed enough iteration such that we ensure we'd hit the internationalization bug (seen starting around iteration 70)?

```suggestion
            Assert.True(result.RunDetails.Max(i => i.ValidationMetrics.RSquared > 0.9));
            
            // Ensure experimentTime allows enough iterations to fully test the internationalization code
            Assert.True(culture == "en-US" || result.RunDetails.Length >= 75);
```

Reply:
Thanks
#Resolved